### PR TITLE
Fix PG::GroupingError according to the sql standard

### PR DESCRIPTION
**Pull request by Zauberstuhl**
Add all id columns to group by statement

### CODE REVIEW

File: lib/vines/storage/sql.rb [hunk 1]
```diff
@@ -39,7 +39,11 @@ class AspectMembership < ActiveRecord::Base
       end
 
       class Contact < ActiveRecord::Base
-        scope :chat_enabled, -> { joins(:aspects).where("aspects.chat_enabled = ?", true) }
+        scope :chat_enabled, -> {
```
Comment (diaspora-code-review) [inline lib/vines/storage/sql.rb]: Use the `lambda` method for multi-line lambdas.

Comment (jhass) [inline lib/vines/storage/sql.rb]: thought I disabled this one.

Comment (Zauberstuhl) [inline lib/vines/storage/sql.rb]: I have to admit that I am not sure what the difference is.

If I recall correctly lambda's are anonymous functions. so what is the difference between:
`scope :chat_enabled, -> {`
and
`scope :chat_enabled, lambda {|chat|`

or was that the reason why you disabled it?

Comment (jhass) [inline lib/vines/storage/sql.rb]: It's a pure style question, there's no semantic difference.

**Pull request merged**